wsi/drm/drmbuffer.cpp

```diff
@@ -240,9 +242,7 @@ void DrmBuffer::SetRecommendedFormat(uint32_t format) {
 
 bool DrmBuffer::CreateFrameBuffer(uint32_t gpu_fd) {
   if (image_.drm_fd_) {
-    ResourceHandle temp;
-    temp.drm_fd_ = image_.drm_fd_;
-    resource_manager_->MarkResourceForDeletion(temp, false);
+    return true;
```

Comment:
We can have cases where native surface being recycled to a different plane might want to allocate fb ignoring alpha channel i.e. Primary plane

Reply:
hmm or this is not relevant anymore as we choose prefered format in drmplane and query that when creating the surface. This might be correct thing to do. I checked quickly but still gem handle -1 and frame buffer creation failures exist :(